static/src/js/zustand/types.ts

```diff
@@ -6,6 +6,16 @@ import {
   TimelineIntervals
 } from '../types/sharedTypes'
 
+export type EarthdataDownloadRedirectSlice = {
+  /** The Earthdata Download Redirect Slice of the store */
+  earthdataDownloadRedirect: {
+    /** The redirect URL for earthdata download */
+    redirect: string
```

Comment:
Let's be more specific with this variable and name it `redirectUrl`, and change the set function to `setRedirectUrl`